mlflow/tracking/client.py

```diff
@@ -446,15 +447,21 @@ def start_trace(
         inputs: Optional[Dict[str, Any]] = None,
         attributes: Optional[Dict[str, str]] = None,
         tags: Optional[Dict[str, str]] = None,
-    ):
+    ) -> MLflowSpanWrapper:
```

Comment:
Can we rename this `MLflowSpanWrapper` to `MlflowSpan`? `Wrapper` sounds like a private/internal class.

Reply:
Yeah but I'm afraid `MLflowSpan` is confusing with `Span` (the one included in `TraceData`), because the latter is MLflow's definition. It is kinda intended to sound internal one, as it is only used when constructing / mutating the spans and never appear once it is published.

Reply:
Got it, thanks for the explanation. I don't have a strong preference for this, to be honest.

Reply:
btw can we lowercase `L`?

Reply:
Ah great catch, will do!